Makefile

```diff
@@ -210,6 +211,8 @@ config.h:
 	echo '/* Default config.h generated by Makefile */' > $@
 	echo '#define HAVE_LIBBZ2 1' >> $@
 	echo '#define HAVE_LIBLZMA 1' >> $@
+	echo '#define HAVE_FSEEKO 1' >> $@
+	echo '#define HAVE_DRAND48 1' >> $@
```

Comment:
The one in `zfopen()` could be changed as is to `fseek` or `rewind`, hence I wrote “that **needs** to use”.  James, there's no need to find something to disagree with in everything I say! 😛